web-ui/src/main/resources/catalog/components/catalog/CatalogService.js

```diff
@@ -748,6 +748,9 @@
 
             images.list[insertFn]({url: s[1], label: s[2]});
```

Comment:
Do you know the reason to use `images.list['push'](...)` here instead of using `images.list.push(...)` like you do below?

Reply:
I see now, because it can use `push` and `unshift`.